docs/manual/customization.md

```diff
@@ -247,23 +247,23 @@ Our [sample deployment](https://raw.githubusercontent.com/foundationdb/fdb-kuber
 
 ### Single-Namespace Mode
 
-To use single-namespace mode, set the `WATCH_NAMESPACE` environment variable for the controller to be the namespace where your FDB clusters will run. It does not have to be the same namespace where the operator is running, though this is generally the simplest way to configure it. When you are running in single-namespace mode, the controller will ignore any clusters you try to create in namespaces other than the one you give it.
+To use single-namespace mode, set the `WATCH_NAMESPACE` environment variable or the command-line option `-watch-namespace` for the controller to be the namespace where your FDB clusters will run. It does not have to be the same namespace where the operator is running, though this is generally the simplest way to configure it. When you are running in single-namespace mode, the controller will ignore any clusters you try to create in namespaces other than the one you give it. If both options are defined, the command line argument will be used.
 
 The advantage of single-namespace mode is that it allows owners of different namespaces to run the operator themselves without needing access to other namespaces that may be managed by other tenants. The only cluster-level configuration it requires is the installation of the CRD. The disadvantage of single-namespace mode is that if you are running multiple namespaces for a single team, each namespace will need its own installation of the controller, which can make it more operationally challenging.
 
 To run the controller in single-namespace mode, you will need to configure the following things:
 
 * A service account for the controller
 * The serviceAccountName field in the controller's pod spec
-* A `WATCH_NAMESPACE` environment variable defined in the controller's pod spec
+* A `WATCH_NAMESPACE` environment variable defined in the controller's pod spec or in the arguments of the container command
 * A Role that grants access to the necessary permissions to all of the resources that the controller manages. See the [sample role](https://raw.githubusercontent.com/FoundationDB/fdb-kubernetes-operator/master/config/samples/deployment/rbac_role.yaml) for the list of those permissions.
 * A RoleBinding that binds that role to the service account for the controller
 
 The sample deployment provides all of this configuration.
 
 ### Global Mode
 
-To use global mode, omit the `WATCH_NAMESPACE` environment variable for the controller. When you are running in global mode, the controller will watch for changes to FDB clusters in all namespaces, and will manage them all through a single instance of the controller.
+To use global mode, omit the `WATCH_NAMESPACE` environment variable or the `-watch-namespace` command line flag for the controller. When you are running in global mode, the controller will watch for changes to FDB clusters in all namespaces, and will manage them all through a single instance of the controller.
```

Comment:
Nitpick: I think this should say you have to omit the variable *and* the flag, rather than the variable *or* the flag.

Reply:
Addressed.